README.md

```diff
@@ -1,33 +1,32 @@
 <h2> Orthoinference: Generating Reactome's Computationally Inferred Reactions and Pathways</h2>
 
-This module has been rewritten from Perl to Java. This first iteration of orthoinference only contains the components of the orthoinference module that pertain to the <a href="https://github.com/reactome/Release/blob/master/scripts/release/orthoinference/infer_events.pl">infer_events.pl</a> script in the Release repository. While this contains the majority of code run for orthoinference, the rewrite of the <a href="https://github.com/reactome/Release/blob/master/scripts/release/orthoinference/remove_unused_PE.pl">remove_unusued_PE.pl</a>, and <a href="https://github.com/reactome/Release/blob/master/scripts/release/updateDisplayName.pl">updateDisplayName.pl</a> modules will need to be completed for this to be considered a completed rewrite.
+This module has been rewritten from Perl to Java. This first iteration of orthoinference only contains the components of the orthoinference module that pertain to the <a href="https://github.com/reactome/Release/blob/master/scripts/release/orthoinference/infer_events.pl">infer_events.pl</a> script in the Release repository. While this contains the majority of code run for orthoinference, the rewrite of the <a href="https://github.com/reactome/Release/blob/master/scripts/release/orthoinference/remove_unused_PE.pl">remove_unused_PE.pl</a>, and <a href="https://github.com/reactome/Release/blob/master/scripts/release/updateDisplayName.pl">updateDisplayName.pl</a> modules will need to be completed for this to be considered a completed rewrite.
```

Comment:
The second line isn't as relevant anymore. The 'remove_unused_PE' wasnt even doing its supposed job, while the updateDisplayName has been successfully baked into Orthoinference. In actuality, all it was generally doing was appending the cellular location to the display name, but that is all done now.

Reply:
Okay, so should the text read:

```
This module has been rewritten from Perl to Java. This first iteration of orthoinference only contains the components of the orthoinference module that pertain to the <a href="https://github.com/reactome/Release/blob/master/scripts/release/orthoinference/infer_events.pl">infer_events.pl</a> script in the Release repository.
```

or just

```
This module has been rewritten from Perl to Java
```

Reply:
I think we can do a big prune here and just use the latter option

Reply:
Done: https://github.com/reactome/release-orthoinference/pull/2/commits/65694f1f91794582d63663a6eff336a58542806a